lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -169,6 +170,27 @@ def install_libraries
           end
         end
 
+        def integrate_test_targets
+          UI.message '- Integrating test targets' do
+            pod_targets_with_native_test_targets = pod_targets.reject { |pt| pt.test_native_targets.empty? }
+            pod_targets_with_native_test_targets.each do |pod_target|
+              pod_target.test_native_targets.each do |native_target|
```

Comment:
This should probably be extracted in a class called `PodTargetIntegrator`, marked it as a task for me.